src/Balance/balance.js

```diff
@@ -71,7 +70,7 @@ export class Balance extends Component {
         const balanceValue = balance[tokenId];
 
         const isEthToken = token.native;
-        const hasBalance = (balanceValue instanceof BigNumber) && balanceValue.gt(0);
+        const hasBalance = balanceValue.isBigNumber && balanceValue.gt(0);
```

Comment:
Should be isBigNumber()? https://github.com/MikeMcl/bignumber.js/issues/111

Reply:
Confusing, but I think I get this right:
- The changelog states `Replace BigNumber.isBigNumber method with isBigNumber prototype property.` @ 4.0.0 https://github.com/MikeMcl/bignumber.js/blob/33e7d0a7cbb9bcd3d3e395d8efe46af17a73da66/CHANGELOG.md#400 ; `.isBigNumber` is set to true for `BigNumber`s.
- The docs/tests all seem to use/recommend `BigNumber.isBigNumber(value)` ; however I think this has been implemented maybe in [6.0.0](https://github.com/MikeMcl/bignumber.js/blob/33e7d0a7cbb9bcd3d3e395d8efe46af17a73da66/CHANGELOG.md#600) because `BigNumber.isBigNumber(...)` is not defined in 4.1.0